vonk/features/multiversion.rst

```diff
@@ -109,13 +109,41 @@ Conformance resources like StructureDefinition and SearchParameter are registere
 Running a single version
 ------------------------
 
-It is currently not possible to run a single version of FHIR in Vonk.
-
-You can only exclude the namespace of the version that you don't need (``Vonk.Fhir.R3`` or ``Vonk.Fhir.R4``) from the :ref:`PipelineOptions <vonk_components_config>` so Vonk does not create unneccessary classes.
+To use only a single version you set the ``Default`` information model in :ref:`information_model` to the version you want to use. In addition, you can exclude the namespace of the version you don't need (``Vonk.Fhir.R3`` or ``Vonk.Fhir.R4``) from the :ref:`PipelineOptions <vonk_components_config>` to disable its use. If you exclude a namespace, make sure to exclude it from all branches.
 
 Running different versions on different endpoints
 -------------------------------------------------
 
-It is currently not possible to run the different versions on different endpoints. 
+To assign endpoints to different versions, create a mapping in :ref:`information_model`. Use the ``Mode`` switch to select either a path or a subdomain mapping, assigning your endpoints in the ``Map`` array. Mapped endpoints will only accept the version you have specified. The web service root ('/') will still accept all supported versions.
```

Comment:
We need to add an example on administration as well, since that is /administration/R4 and not /R4/administration.
Or more in general: the path postfix is postfixed to the PipelineOptions:Branch:Path.